Jint/Runtime/Debugger/DebugScope.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Jint.Native;
+
+namespace Jint.Runtime.Debugger
+{
+    /// <summary>
+    /// Dictionary of bindings for a single scope in the scope chain
+    /// </summary>
+    public class DebugScope : IReadOnlyDictionary<string, JsValue>
+    {
+        private IReadOnlyDictionary<string, JsValue> variables;
+
+        /// <summary>
+        /// The type of scope. Scope types are the same as defined by Chrome devtools protocol.
+        /// </summary>
+        public DebugScopeType ScopeType { get; }
```

Comment:
ideally order would be fields, constructor, properties, methods

Reply:
I'm a slob about order of members. Going through and fixing. 🙂